go.mod

```diff
@@ -52,10 +52,10 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20190318030020-c3a204f8e965
 	github.com/unrolled/render v1.0.1
 	github.com/urfave/negroni v0.3.0
-	go.etcd.io/etcd/api/v3 v3.5.15
-	go.etcd.io/etcd/client/pkg/v3 v3.5.15
-	go.etcd.io/etcd/client/v3 v3.5.15
-	go.etcd.io/etcd/server/v3 v3.5.15
+	go.etcd.io/etcd/api/v3 v3.5.20
```

Comment:
Does it mean we should upgrade PD to this version first? So we need to wait for at least one release version, then we can upgrade to v3.6?

Reply:
Maybe we should wait a LTS version 🤔